docker/nix-build.sh

```diff
@@ -4,6 +4,13 @@ set -euo pipefail
 SPAWN_SHELL=${SPAWN_SHELL:-false}
```

Comment:
Please move this file and `Dockerfile_build` to `nix` folder. Please update `build.md` accordingly.

Reply:
Moved.